feat: allow tres components written in kebab-case (#1015)
Tresjs/tres: docs/content/1.getting-started/1.index.md

```

This approach **reduces the learning curve** significantly while preserving all the power and flexibility of Three.js.

### Component Name Casing

> Throughout the guide, we are using PascalCase names when registering components. You can find more of the reasoning behind this in the official [Vue.js docs](https://vuejs.org/guide/components/registration.html#component-name-casing) This is because:

Reviewer: ```suggestion
Throughout the guide, we are using PascalCase names when registering components. You can find more of the reasoning behind this in the official [Vue.js docs](https://vuejs.org/guide/components/registration.html#component-name-casing).

> Luckily, Tres supports resolving kebab-case tags to components registered using PascalCase. This means a component registered as MyComponent can be referenced inside a Vue template (or inside an HTML element rendered by Vue) via both <MyComponent> and <my-component>. If you want to use the kebab-case notation in combination with the TresJS eslint config, you have to deactivate the following rule:
> `vue/component-name-in-template-casing` in your eslint configuration.

Reviewer: ```suggestion
Luckily, Tres supports resolving kebab-case tags to components registered using PascalCase. This means a component registered as MyComponent can be referenced inside a Vue template (or inside an HTML element rendered by Vue) via both <MyComponent> and <my-component>. If you want to use the kebab-case notation in combination with the TresJS eslint config, you should deactivate the rule 
`vue/component-name-in-template-casing` in your eslint configuration.
```

### Component Name Casing

Throughout the guide, we are using PascalCase names when registering components.

> ::read-more

Reviewer 2: @colinscz You can use the readmore by passing the link like:

```
::read-more{to="https://vuejs.org/guide/components/registration.html#component-name-casing"}
```
